### ISSUE TITLE
[pfsense_openvpn_client] Add support for additional values in `digest` field

### ISSUE THREAD

**Issue by samuel-emrys** [pfsensible/core#134]
**Is your feature request related to a problem? Please describe.**
The only values that can currently be passed to the `digest` field are `SHA1` and `SHA256`, as per plugins/module_utils/openvpn_client.py:41:

```python
    digest=dict(default='SHA256', required=False, choices=['SHA256', 'SHA1']),
```

This populates the digest xml field:

<digest>SHA256</digest>

However, pfSense also supports substantially more than these two values, including `SHA224`, `SHA512`, `SHA384`, `MD5`, among others.

**Describe the solution you'd like**
Entries for each of the different options in the options list

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by opoplawski** [comment]
Yeah, I'm not sure how  best to handle this.  pfSesnse generates the options dynamically from the output of `openvpn --show-digests` and `openssl list --digest-algorithms`, so any list in the module documentation will likely be incomplete or need updating.  Maybe just making it a text field and validating at run time with the same output would be the way to go.
